src/web-ui/src/public/Footer.vue

```diff
@@ -1,6 +1,19 @@
 <template>
   <div class="footer">
     <div class="container">
+      <div class="row" v-if="user">
+        <form class="mt-2 mb-2 mx-auto text-center">
+          <div class="form-group mb-2 mr-sm-2">
+            <label for="sms-alerts">Join Text Alerts and get 20% off</label>
+            <div class="input-group">
+              <input type="text" class="form-control" id="sms-alerts" placeholder="Enter cell phone number" aria-describedby="basic-addon2">
+              <div class="input-group-append">
+                <button type="button" class="btn btn-outline-secondary mb-2" v-on:click="verifyAndUpdateUserPhoneNumber">Submit</button>
+              </div>
```

Comment:
I assume the messaging for "messaging and data rates may apply..." will be added at some point?

Reply:
Yes, I need to update this section with Kaz's changes. I will update it with the rest of the changes today.